tests/xmltester/tests/general/TestDensify.xml

```diff
@@ -16,39 +16,47 @@
+  (10 50, 17.5 50, 25 50, 32.5 50, 40 50, 47.5 50, 55 50, 62.5 50, 70 50))
 	</op></test>
 </case>
 
+<case>
+  <desc>A - simple polygon</desc>
+  <a>    POLYGON ((0 0, 0 20, 20 20, 20 0, 0 0))  </a>
+<test><op name="densify" arg1='A'  arg2='10.0'>
+  POLYGON ((0 0, 0 10, 0 20, 10 20, 20 20, 20 10, 20 0, 10 0, 0 0))
+  	</op></test>
+</case>
 
 <case>
   <desc>A - polygon with hole</desc>
   <a>    POLYGON ((0 0, 0 70, 70 70, 70 0, 0 0), (10 10, 10 60, 60 60, 10 10))  </a>
-<test><op name="densify" arg1='A'  arg2='10.0'>    
-POLYGON ((0 0, 0 8.75, 0 17.5, 0 26.25, 0 35, 0 43.75, 0 52.5, 0 61.25, 0 70, 8.75 70, 17.5 70, 26.25 70, 35 70, 43.75 70, 52.5 70, 61.25 70, 70 70, 70 61.25, 70 52.5, 70 43.75, 70 35, 70 26.25, 70 17.5, 70 8.75, 70 0, 61.25 0, 52.5 0, 43.75 0, 35 0, 26.25 0, 17.5 0, 8.75 0, 0 0), 
-  (10 10, 16.25 16.25, 22.5 22.5, 28.75 28.75, 35 35, 41.25 41.25, 47.5 47.5, 53.75 53.75, 60 60, 51.66666666666667 60, 43.333333333333336 60, 35 60, 26.666666666666668 60, 18.333333333333336 60, 10 60, 10 51.66666666666667, 10 43.333333333333336, 10 35, 10 26.666666666666668, 10 18.333333333333336, 10 10))
+<test><op name="densify" arg1='A'  arg2='10.0'>
+  POLYGON ((0 0, 0 10, 0 20, 0 30, 0 40, 0 50, 0 60, 0 70, 10 70, 20 70, 30 70, 40 70, 50 70, 60 70, 70 70, 70 60, 70 50, 70 40, 70 30, 70 20, 70 10, 70 0, 60 0, 50 0, 40 0, 30 0, 20 0, 10 0, 0 0),
+  (10 10, 16.25 16.25, 22.5 22.5, 28.75 28.75, 35 35, 41.25 41.25, 47.5 47.5, 53.75 53.75, 60 60, 50 60, 40 60, 30 60, 20 60, 10 60, 10 50, 10 40, 10 30, 10 20, 10 10))
   	</op></test>
 </case>
 
 <case>
   <desc>mA - multipolygon</desc>
-  <a>    MULTIPOLYGON (((0 0, 0 70, 70 70, 70 0, 0 0), 
-  (10 10, 10 60, 60 60, 10 10)), 
+  <a>    MULTIPOLYGON (((0 0, 0 70, 70 70, 70 0, 0 0),
+  (10 10, 10 60, 60 60, 10 10)),
   ((80 110, 80 70, 120 70, 120 110, 80 110)))  </a>
-<test><op name="densify" arg1='A'  arg2='10.0'>    
-MULTIPOLYGON (((80 110, 88 110, 96 110, 104 110, 112 110, 120 110, 120 102, 120 94, 120 86, 120 78, 120 70, 112 70, 104 70, 96 70, 88 70, 80 70, 80 78, 80 86, 80 94, 80 102, 80 110)), 
-  ((0 0, 0 8.75, 0 17.5, 0 26.25, 0 35, 0 43.75, 0 52.5, 0 61.25, 0 70, 8.75 70, 17.5 70, 26.25 70, 35 70, 43.75 70, 52.5 70, 61.25 70, 70 70, 70 61.25, 70 52.5, 70 43.75, 70 35, 70 26.25, 70 17.5, 70 8.75, 70 0, 61.25 0, 52.5 0, 43.75 0, 35 0, 26.25 0, 17.5 0, 8.75 0, 0 0), 
-    (10 10, 16.25 16.25, 22.5 22.5, 28.75 28.75, 35 35, 41.25 41.25, 47.5 47.5, 53.75 53.75, 60 60, 51.66666666666667 60, 43.333333333333336 60, 35 60, 26.666666666666668 60, 18.333333333333336 60, 10 60, 10 51.66666666666667, 10 43.333333333333336, 10 35, 10 26.666666666666668, 10 18.333333333333336, 10 10)))      	</op></test>
+<test><op name="densify" arg1='A'  arg2='10.0'>
+  MULTIPOLYGON (((80 70, 80 80, 80 90, 80 100, 80 110, 90 110, 100 110, 110 110, 120 110, 120 100, 120 90, 120 80, 120 70, 110 70, 100 70, 90 70, 80 70)),
```

Comment:
Note: first ring normalized (rotated) a bit differently than previous version.  The primary change on this ring is that segments are now `10` long, as expected from the tolerance.

Reply:
If you are open to it, I'd like to simplify the coordinates of these densify tests to make it easier to spot and verify the expected output.

Reply:
Excellent idea.